pkg/theia/commands/anomaly_detection_retrieve.go

```diff
@@ -105,13 +104,37 @@ func throughputAnomalyDetectionRetrieve(cmd *cobra.Command, args []string) error
 		}
 		return nil
 	} else {
-		w := tabwriter.NewWriter(os.Stdout, 15, 8, 1, '\t', tabwriter.AlignRight)
-		fmt.Fprintf(w, "id\tsourceIP\tsourceTransportPort\tdestinationIP\tdestinationTransportPort\tflowStartSeconds\tflowEndSeconds\tthroughput\talgoCalc\tanomaly\n")
-		for _, p := range tad.Stats {
-			fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%v\t%v\t%v\t%v\t%v\t%v\n", p.Id, p.SourceIP, p.SourceTransportPort, p.DestinationIP, p.DestinationTransportPort, p.FlowStartSeconds, p.FlowEndSeconds, p.Throughput, p.AlgoCalc, p.Anomaly)
+		var result [][]string
+		switch tad.Stats[0].AggType {
+		case "e2e":
+			result = append(result, []string{"id", "sourceIP", "sourceTransportPort", "destinationIP", "destinationTransportPort", "flowStartSeconds", "flowEndSeconds", "throughput", "aggType", "algoType", "algoCalc", "anomaly"})
+			for _, p := range tad.Stats {
+				result = append(result, []string{p.Id, p.SourceIP, p.SourceTransportPort, p.DestinationIP, p.DestinationTransportPort, p.FlowStartSeconds, p.FlowEndSeconds, p.Throughput, p.AggType, p.AlgoType, p.AlgoCalc, p.Anomaly})
+			}
+		case "pod":
+			if tad.Stats[0].PodLabels != "" {
```

Comment:
A similiar case here, our default output table will be the same as the podLabels one